blockchain/client.go

```diff
@@ -197,6 +199,11 @@ func (c *Client) ListenEvents(
 					return err
 				}
 			}
+		// Watchdog for the websocket. It silently hangs sometimes with no error nor new events. In
+		// all Cere blockchain runtimes we have `pallet-timestamp` which makes at least one event
+		// (System.ExtrinsicSuccess for the timestamp.set extrinsic) per block.
+		case <-time.After(60 * time.Second):
```

Comment:
might be either move to constant or make it configurable in client creation method and set it as a default (still in constant)?

Reply:
Updated, I made it a constant.